server/docker-base/template.local.properties

```diff
@@ -1 +1 @@
-sdk.dir={{android-sdk-root}}
+sdk.dir={{sdk-dir}}
```

Comment:
The name of this file is a bit anonymous.
Perhaps add some "gradle" to the name?